### PR TITLE
Fix flaky integration test

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Enable integration test and update it according latest changes in SAVE

### CODE REVIEW

**Comment by kgevorkyan** [comment]
Seems main problem are gone after https://github.com/diktat-static-analysis/save/pull/310

From the 6 CI executions there was only one failure on MacOS, on other OS all worked fine every execution

File: save-cli/src/jvmTest/kotlin/org/cqfn/save/cli/GeneralTest.kt [hunk 1]
```diff
@@ -27,9 +32,16 @@ import kotlinx.serialization.json.Json
 @OptIn(ExperimentalSerializationApi::class)
 class GeneralTest {
     private val fs = FileSystem.SYSTEM
+    private val json = Json {
```
Comment (petertrr) [inline save-cli/src/jvmTest/kotlin/org/cqfn/save/cli/GeneralTest.kt:35]: Can we reuse this `Json` from somewhere else? We could have `save-reporters` as a test dependency here (IIRC, there is the same polymorphic serializer there?)

**Comment by petertrr** [comment]
> Seems main problem are gone after #310
> 
> From the 6 CI executions there was only one failure on MacOS, on other OS all worked fine every execution

Were the errors that you faced same as in #311 ?

File: save-cli/src/jvmTest/kotlin/org/cqfn/save/cli/GeneralTest.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-cli/src/jvmTest/kotlin/org/cqfn/save/cli/GeneralTest.kt]: Refactored

**Comment by kgevorkyan** [comment]
> > Seems main problem are gone after #310
> > From the 6 CI executions there was only one failure on MacOS, on other OS all worked fine every execution
> 
> Were the errors that you faced same as in #311 ?

Actually not, the exception were like `test resources were not found`, as if test files were removed
At the same moment several tests for plugins were failed too

The reason for now is unclear and such situation were not reproduced

**Review by petertrr — APPROVED** [review]

**Pull request merged**